lib/ansible/modules/packaging/os/pacman.py

```diff
@@ -433,6 +433,7 @@ def main():
     )
 
     pacman_path = module.get_bin_path('pacman', True)
+    module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C', LC_CTYPE='C')
```

Comment:
I think the following should be sufficient:
```
module.run_command_environ_update = dict(LC_ALL='C')
```
because https://pubs.opengroup.org/onlinepubs/7908799/xbd/envvar.html :
> The value of the LC_ALL environment variable has precedence over any of the other environment variables starting with LC_ (LC_COLLATE, LC_CTYPE, LC_MESSAGES, LC_MONETARY, LC_NUMERIC, LC_TIME) and the LANG environment variable.

Reply:
Indeed. Good catch. Fixed.
There are some other modules that could benefit from this simplification as well.